gits/test/helper.py

```diff
@@ -347,33 +347,38 @@ def _check_cap_csr(self, reg):
         self.assertEqual(top - 1, term._top_most)
 
     @reset_after_executing
-    def _check_cap_dl1(self, s, pos):
-        """A helper that checks the `_cap_dl1` method.
-
-        The ``s`` argument is a test string to be put on the screen.
-        The ``pos`` argument must be a tuple or list of coordinates ``(x, y)``
-        of the location where you want the string to be.
+    def _check_cap_dl(self, n, lines_positions):
+        """A helper that checks the `_cap_dl` method.
+
+        The ``n`` argument is a number of lines to be deleted from the screen.
+        The ``lines_positions`` argument must be a list of tuples
+        ``(line, pos)``, where ``line`` is a test string to be put on the
+        screen, ``pos`` is a tuple or list of coordinates ``(x, y)`` of the
+        location where you want the string to be.
         """
-        cur_x, cur_y = pos
-
-        self._put_string(s, pos)
-
         term = self._terminal
-        term._cur_x, term._cur_y = pos
 
-        term._cap_dl1()
+        for line, pos in lines_positions:
+            self._put_string(line, pos)
 
-        self.assertEqual(cur_x, term._cur_x)
+        # There is a thought that the `dl` capability will be used together
+        # with the `home` capability to actually delete a line.
```

Comment:
We have to put the cursor on the line that is going to be deleted. In this particular case, the `home` capability does what we really want, but it doesn't mean that the capabilities are always used together. Moreover, it's quite difficult to find a program which is used `dl`. As far as I know, `vi`, `nano`, `top`, `htop` and `mc` don't use it.